packages/schemas/tables/settings.sql

```diff
@@ -0,0 +1,13 @@
+create type tenant_region_type as enum ('china_mainland', 'china_hongkong', 'china_taiwan', 'united_states', 'japan');
+
+create table settings (
+    tenant_id varchar(24) not null,
+    tenant_name text not null,
+    tenant_region tenant_region_type not null default('china_mainland'),
+    skip_get_started boolean not null default('true'),
```

Comment:
maybe `application_skip_get_started`? or we can simply set up a field named `admin_console` or something with JSONB value which indicates the persistent customization data of Admin Console.

note we assume Logto can run WITHOUT Admin Console. it'll make more sense if we aggregate data in a single field.